Source/FolioReaderKit.swift

```diff
@@ -169,8 +173,12 @@ extension FolioReader {
         Bool = true) {
         let readerContainer = FolioReaderContainer(withConfig: config, folioReader: self, epubPath: epubPath, unzipPath: unzipPath, decryptionKey: decryptionKey, removeEpub: shouldRemoveEpub)
         self.readerContainer = readerContainer
+        let folioReaderTransitionDelegate = FolioReaderTransitionDelegate()
         readerContainer.modalPresentationStyle = .fullScreen
-        parentViewController.present(readerContainer, animated: animated, completion: nil)
+        readerContainer.transitioningDelegate = folioReaderTransitionDelegate
```

Comment:
can we combine `176` and `178` into one line? 
`readerContainer.transitioningDelegate = FolioReaderTransitionDelegate()`

Reply:
lines combined